test/help-init/expected-stdout

```diff
@@ -19,7 +19,7 @@ Usage:
   %s init [flags]
 
 Flags:
-  -b, --allowed-branches string   comma separated IDs or name globs, use "*" for all (default "main")
+  -b, --branches string           comma separated IDs or name globs, use "*" for all (default "main")
```

Comment:
`--allowed-branches ` -> `--branches` aby to bolo rovnako u vsetkych prikazov, co potrebuju zadat `branches`.
Podobne ako mame inde zase `configs`.